### ISSUE TITLE
ACCheckbox/AutoConnectCheckbox doesn't work for esp8266 from AutoConnect 1.3.7

### ISSUE THREAD

**Issue by Evgen2** [Hieromon/AutoConnect#561]
[CheckBoxIssue.zip](https://github.com/Hieromon/AutoConnect/files/10456381/CheckBoxIssue.zip)
Construction like 

```
ACText(Caption, "Hello, world", "", "", AC_Tag_DIV);
ACCheckbox(ChB1, "chb", "chb");
//or AutoConnectCheckbox ChB1("ChB1","1", "m", false, AC_Behind , AC_Tag_BR);
ACSubmit(Apply, "Apply", HELLO_URI);
// AutoConnectAux for the custom Web page.
AutoConnectAux helloPage(HELLO_URI, "Hello", true, { Caption, ChB1,  Apply });

```

work well for esp8266 with AutoConnect 1.3.3 and lead to  Exeption(3) with 1.3.7 or 1.41
Code for esp32 work
![ex3](https://user-images.githubusercontent.com/11079227/213436348-53bc04b5-d5e7-4b82-ad83-a5b65d7098ca.jpg)

**Comment by Hieromon** [comment]
I have reproduced the issue. It is AutoConnect bug. The readout from PROGMEM is missing.

I will fix it soon and release as AutoConnect v1.4.2. You can evaluate the fix ​before the release with the patches branch.

Thank you for reporting the issue.

**Comment by Hieromon** [comment]
Merged #575
